ast.hpp

```diff
@@ -53,6 +53,13 @@
 namespace Sass {
   using namespace std;
 
+  template <typename T>
```

Comment:
Please add the correct attribution to boost.
https://github.com/boostorg/functional/blob/1bf379ab601eb52c97908cc2cc141fd6026bd708/include/boost/functional/hash/hash.hpp#L209-L213